basic/list2.py

```diff
@@ -13,17 +13,23 @@
 # so [1, 2, 2, 3] returns [1, 2, 3]. You may create a new list or
 # modify the passed in list.
 def remove_adjacent(nums):
-    # +++your code here+++
-    return
+	elemek={}
+	for num in nums:
+		if(num not in elemek): elemek[num]=[num]
+	uj=[]
+	for i in elemek:
+		uj.append(i)	
+    return uj
 
 
 # E. Given two lists sorted in increasing order, create and return a merged
 # list of all the elements in sorted order. You may modify the passed in lists.
 # Ideally, the solution should work in "linear" time, making a single
 # pass of both lists.
 def linear_merge(list1, list2):
-    # +++your code here+++
```

Comment:
Nem igazan. Nem hasznalod az extra infot, hogy mindket list rendezett.  igy nem is lesz O(n)